source/blog/2016-12-12-part2-install-origin-on-f25-atomic-host.html.md

```diff
@@ -0,0 +1,353 @@
+---
+title: 'Installing an OpenShift Origin Cluster on Fedora 25 Atomic Host: Part 2'
+author: dustymabe
+date: 2016-12-12 15:23:32 UTC
+tags: fedora, atomic, origin, openshift, kubernetes
+published: true
+comments: true
+---
+
+Introduction
+============
+
+In [part 1](http://www.projectatomic.io/blog/2016/12/part1-install-origin-on-f25-atomic-host/)
+of this series, we used the [OpenShift Ansible
+Installer](https://github.com/openshift/openshift-ansible) to install
+[Openshift Origin](https://github.com/openshift/origin) on three servers
+that were running [Fedora 25 Atomic
+Host](https://getfedora.org/en/atomic/). The three machines we'll be using
```

Comment:
I would move this whole chunk ("The three machines..." + the table) to under "The Installed Origin Cluster". It seems weird to drop those details while still in the intro.